File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/runreports/reportcategory/ReportCategoryMvpView.java
package com.mifos.mifosxdroid.online.runreports.reportcategory;

import com.mifos.mifosxdroid.base.MvpView;
import com.mifos.objects.runreports.client.ClientReportTypeItem;

import java.util.List;

/**
 * Created by Tarun on 03-08-17.
 */

public interface ReportCategoryMvpView extends MvpView {

    void showError(String error);

    void showReportCategories(List<ClientReportTypeItem> reportTypes);
}

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/runreports/reportcategory/ReportCategoryMvpView.kt
package com.mifos.mifosxdroid.online.runreports.reportcategory

import com.mifos.mifosxdroid.base.MvpView
import com.mifos.objects.runreports.client.ClientReportTypeItem

/**
 * Created by Tarun on 03-08-17.
 */
interface ReportCategoryMvpView : MvpView {
    fun showError(error: String)

    fun showReportCategories(reportTypes: List<ClientReportTypeItem>)
}